Sources/MapboxNavigation/StepsViewControllerDelegate.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+/**
+ `StepsViewControllerDelegate` provides methods for user interactions in a `StepsViewController`.
+ */
+public protocol StepsViewControllerDelegate: AnyObject {
```

Comment:
Speaking of the step table, what does it look like when you tap or swipe down on the top banner? Does it still get slotted into the space between the top and bottom banners as it does now? If so, we can tackle improving that layout as tail work. There isn’t a very natural place for the step table in this landscape orientation, but maybe presenting it as a modal view controller would be an improvement.

Reply:
Yep, it's currently fit between leading and trailing anchors of top and bottom banners:

![Simulator Screen Shot - iPod touch (7th generation) - 2022-01-20 at 11 22 31](https://user-images.githubusercontent.com/1496498/150407840-5969493c-4e59-4bab-ae54-5feefb2bc4b6.png)

It doesn't look too bad, but more space would improve things, I'll think about possible improvements.